Features/template

ohjonah opened the pull request.

ohjonah committed:
Changed default branch to master

ohjonah committed:
Hero Image and Text Overlay

Added Hero Image and overlay for box and text

ohjonah committed:
Add padding and transparent overlay

ohjonah committed:
Meta Tag for Viewport

ohjonah committed:
Added jQ script and mobile nav

Added jQuery script file to utilize jQuery in javascript files and enabled nav to be shown on click

ohjonah committed:
removes unused jQ method

removed toggle since it wasn't toggling anything

ohjonah committed:
Added anchor tags

Added anchor tags to link to github and linkedin on icons and nav

ohjonah committed:
New portfolioProjects JS file

create new portfolioProjects.js to store portfolio project data. this is linked in index.html

ohjonah committed:
Projects Raw Data and Instantiated Projects

Filled in data for raw project data and instantiated project data that is pushed to an array in app.js

ohjonah committed:
Added Project Template Section

created project template section and div text overlay that will be filled dynamically using jQ and styled with CSS

ohjonah committed:
Project Styling Added

Added project div styling so that I can see results of clone

ohjonah committed:
Populated Dom with Projec Title

Added forEach statement to populate DOM with instantiated projects

musflood commented on css/modules.css:
```diff
@@ -17,3 +21,77 @@
+  text-decoration: underline;
+}
+
+.template {
```
You should not apply these styles to the _template_ class because the template has `display: none;`. Instead you should apply them to the _articles_, that way, when the projects are copied from the template, all your styles will automatically be applied.

ohjonah replied:
changed!

ohjonah committed:
Mobile Styling Added

changed css to target element article instead of .template. Text align center for mobile
